internal/util.go

```diff
@@ -43,8 +43,8 @@ func EscapeColumnName(s string) string {
 	upperedColumnName := strings.ToUpper(snaker.ForceCamelIdentifier(s))
 	if _, ok := reservedKeywords[upperedColumnName]; ok {
 		// return surrounded snaker.SnakeToCamelIdentifier(s) with backquotes if reserved keyword
-		return fmt.Sprintf("`%s`", snaker.ForceCamelIdentifier(s))
```

Comment:
Please fix the comment above and this is no longer need to use upperedColumnName for the check.